gradoop-flink/src/main/java/org/gradoop/flink/model/impl/functions/epgm/Id.java

```diff
@@ -18,16 +18,16 @@
 import org.apache.flink.api.common.functions.MapFunction;
 import org.apache.flink.api.java.functions.FunctionAnnotation;
 import org.apache.flink.api.java.functions.KeySelector;
+import org.gradoop.common.model.api.entities.EPGMElement;
 import org.gradoop.common.model.impl.id.GradoopId;
-import org.gradoop.common.model.impl.pojo.Element;
 
 /**
  * element => elementId
  *
  * @param <EL> element type
  */
 @FunctionAnnotation.ForwardedFields("id->*")
-public class Id<EL extends Element>
+public class Id<EL extends EPGMElement>
```

Comment:
Why the restriction? This makes the function unusable on any other potential subclass of Element, without any clear advantage.

Reply:
Switch to EPGMElement is no restriction but an generalization since EPGMElement is the Interface of all Graph Elements.

Reply:
Resolve it if you agree :)